src/plugins/data/common/field_formats/types.ts

```diff
@@ -39,8 +43,8 @@ export type FieldFormatConvertFunction = HtmlContextTypeConvert | TextContextTyp
 
 /** @internal **/
 export interface FieldFormatConvert {
-  text: TextContextTypeConvert;
-  html: HtmlContextTypeConvert;
+  [TEXT_CONTENT_TYPE]: TextContextTypeConvert;
```

Comment:
it was 'text' and 'html' here to avoid of circular reference. 

content_types.ts  imports this file

Reply:
Thanks. Reverted.